packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -350,7 +350,8 @@ export const prepareVault = (baggage, makeRecorderKit, zcf) => {
           }
           const { phase } = state;
           const uiState = facets.helper.getStateSnapshot(phase);
-          trace('updateUiState', state.idInManager, uiState);
+          const brand = facets.helper.collateralBrand();
+          trace('updateUiState', state.idInManager, uiState, brand);
```

Comment:
oh yeah, the unique id before wasn't really unique (over the factory)

are there any other vault-level traces that need these?

Reply:
I looked through vaults and didn't add details when there were consecutive `traces` without an intervening `await`.